apps/condo/domains/news/gql.js

```diff
@@ -6,12 +6,14 @@
 
 const { generateGqlQueries } = require('@open-condo/codegen/generate.gql')
 
+const { ADDRESS_META_SUBFIELDS_QUERY_LIST } = require('@condo/domains/property/schema/fields/AddressMetaField')
```

Comment:
Do you really need all of this huge amount of fields?
<img width="1427" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/d6d52297-5d3a-4037-ae74-37328db5a5a8">

Reply:
done btw
it may be not noticeable but I replaced constant ADDRESS_META_SUBFIELDS_QUERY_LIST to ADDRESS_META_SUBFIELDS_TABLE_LIST

its necessary because the fields from it are used in property table cell render